### DISCUSSION TITLE
🚧 New Documentation Site (Work in Progress, Not Official)

### DISCUSSION THREAD

**Discussion by salmoneatenbybear** [salmoneatenbybear/circuitverse-docs#1] [Announcements]
Hi CircuitVerse Community,  

I’ve been working on a **new Docusaurus-powered documentation site** to improve SEO and enhance the user experience. Please note:  

- **This is not the official CircuitVerse documentation.**  
- It’s a **work in progress**, created as a personal initiative to explore better documentation options for the community. 

### What’s New?
- **Improved SEO** for better discoverability.  
- **Powerful Algolia search** for faster, more intuitive content finding.  
- A **Contributors Page** to honor everyone who’s contributed to CircuitVerse.  

### How You Can Help
I’d love to hear your feedback! Please let me know if:  
- You find any bugs 🐛.  
- You have suggestions for improvements ✨.  
- You think of any additional features or enhancements.  

Feel free to comment below or open an issue in the GitHub repository. Your input is invaluable!  

Thank you for your time and support.

**Comment by salmoneatenbybear** [comment]
Please be aware of above info. 



@SharonIV0x86

@ManishG04

I am a contributor to Circuitverse not a maintainer or mentor and I was working on an issue to create a new documentation site as it was needed by the maintainers of circuitverse.

**Reply by ManishG04** [reply]
ok, got it

**Reply by salmoneatenbybear** [reply]
ALSO please include screenshots or casts if things are UI related, helps verifying viability of issues

**Reply by salmoneatenbybear** [reply]
@ManishG04 @SharonIV0x86

**Comment by salmoneatenbybear** [comment]
Shoutout to @SharonIV0x86 for his contributions to this repo ! 🤟🤟

**Comment by salmoneatenbybear** [comment]
shoutout to @Jay2006sawant for his contributions 💯💯💯

**Comment by salmoneatenbybear** [comment]
Shoutout to @aniketchawardol for his contributions to improve the CircuitVerse documentation.

🎉💯🔥

**Comment by aniketchawardol** [comment]


**Reply by salmoneatenbybear** [reply]
Which task ?

**Reply by salmoneatenbybear** [reply]
The new site is already built, the task is done but, no response from maintainers about it

**Comment by salmoneatenbybear** [comment]
@Asrani-Aman, @gr455, @tachyons, @vedant-jain03, @VaibhavUpreti @Arnabdaz

would love your views on this site i built to improve seo performance...

[circuitverse-docs.netlify.app/](https://circuitverse-docs.netlify.app/)

lot of contributors also actively started to make this site better too. 

this was done in order to solve the issue : https://github.com/CircuitVerse/CircuitVerseDocs/issues/383

**Comment by salmoneatenbybear** [comment]
@SharonIV0x86 

Pls could you join here https://circuitverse.org/slack
